taskfiles/rp.yml

```diff
@@ -61,17 +61,22 @@ tasks:
       PATH={{.LLVM_INSTALL_PATH}}/bin:$PATH
       ninja -C '{{.BUILD_DIR}}' -j'{{div .TOTAL_PHYSICAL_MEMORY .GB_PER_BUILD_CORE}}'
 
+  set-aio-max:
+    desc: set minimum required value for fs.aio-max-nr sysctl option
+    vars:
+      MIN_REQUIRED_AIO_MAX: 1048576
+    cmds:
+    - sudo sysctl -w fs.aio-max-nr={{.MIN_REQUIRED_AIO_MAX}}
+    status:
+    - test {{.MIN_REQUIRED_AIO_MAX}} -le $(sysctl -nb fs.aio-max-nr)
+
   test:
     desc: run redpanda unit tests
     vars:
       CI: '{{ default "0" .CI}}'
       CTEST_ARGS: '{{default `-R ".*_rp(unit|bench|int)$"` .CTEST_ARGS}}'
-      MIN_REQUIRED_AIO_MAX: 1048576
     dir: '{{.BUILD_DIR}}'
+    deps:
```

Comment:
k8s also needs aio @RafalKorepta was having issues here. can you add this as part of the k8s test dependency

Reply:
the k8s tests are currently automated with the `src/go/k8s/Makefile` that needs to be ported to Task (I opened https://github.com/vectorizedio/redpanda/issues/952)